utils/build/docker/dotnet/parametric/DuckTypes/IDuckContextExtractor.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace ApmTestClient.DuckTypes;
+
+public interface IDuckContextExtractor
```

Comment:
Do you still need this file? Maybe I missed it, but I didn't see this being used

Reply:
Good catch! I cleaned up all the unused duck types and only `IDuckSpanContext` was left.

This was left over from earlier experiments, including I think removing most the usage of reflection in the non-OTel tests. I might come back and to that again later...